src/Features/Core/Portable/GenerateMember/AbstractGenerateMemberService.cs

```diff
@@ -90,7 +90,7 @@ protected AbstractGenerateMemberService()
 
             var syntaxFacts = document.Project.LanguageServices.GetService<ISyntaxFactsService>();
             var semanticModel = document.SemanticModel;
-            var isMemberAccessExpression = syntaxFacts.IsMemberAccessExpression(simpleNameOrMemberAccessExpression);
+            var isMemberAccessExpression = syntaxFacts.IsSimpleMemberAccessExpression(simpleNameOrMemberAccessExpression);
```

Comment:
Yes.  It was poorly names.  I ended up adding IsSimpleMemberAccessExpression and then i realized it was the exact same as the existin gmethod.